### DISCUSSION TITLE
Sioyek not reloading automatically on macOS

### DISCUSSION THREAD

**Discussion by Ruboss39** [ahrm/sioyek#966] [Q&A]
Hi,

When I use Sioyek as a pdf-viewer with Vimtex, Sioyek sometimes does not reload the pdf automatically. In such a case, I switch to the Sioyek window and type in :reload, to manually reload the PDF. 

Usually, when I have recently started Sioyek, Sioyek automatically reloads the PDF when the PDF is changed/updated.  After 5 minutes or so, I need to reload manually. Is there a setting I am missing or so?

Also, what does :reload_config do?

**Comment by ahrm** [comment]
Is this on the main branch or the development branch? I think this is probably fixed in the development branch. `reload_config` reloads the `prefs_user` and `keys_user` config files. We usually reload the automatically when they are changed but this command is for cases where automatic reloading doesn't work for whatever reason.

**Reply by Ruboss39** [reply]
This is on the main branch.

**Reply by daxmate** [reply]
I had the same trouble and solved it with reinstall sioyek.
